tests/catalog/test_base.py

```diff
@@ -312,7 +312,7 @@ def test_rename_table(catalog: InMemoryCatalog) -> None:
     assert table._identifier == Catalog.identifier_to_tuple(new_table)
 
     # And
-    assert ("new", "namespace") in catalog.list_namespaces()
+    assert ("new",) in catalog.list_namespaces()
```

Comment:
👍 this is the right behavior, only return the top level namespace
https://github.com/apache/iceberg/blob/41b458b7022c7b0cd78eeca9102392db7889d3c9/core/src/test/java/org/apache/iceberg/jdbc/TestJdbcCatalog.java#L770-L772

Reply:
although looking at the context of the test, i think its testing for existence of `("new", "namespace")`

so maybe something like this
```suggestion
    assert ("new", "namespace") in catalog.list_namespaces("new")
```

Reply:
good point! I'll change the assertion from using `list_namespaces()` to `_namespace_exists()`